index.html

```diff
@@ -11881,13 +11883,14 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
+				<p>Since different users have different needs, it is up to the user to tweak their assistive technologies' response to a live region. Assistive technologies might choose to implement increasing and decreasing levels of granularity so that the user can exercise control over queues and interruptions.</p>
+				<p>When the [=ARIA/property=] is not set on an <a>object</a> that needs to send updates, the priority level is the value of the nearest ancestor that sets the <pref>aria-live</pref> attribute.</p>
+				<p>The <pref>aria-live</pref> attribute is the primary determination for the order of presentation of changes to live regions. Implementations will also consider the default level of priority in a <a>role</a> when the <pref>aria-live</pref> attribute is not set in the ancestor chain (e.g., <rref>log</rref> changes are <code>polite</code> by default). Modifications to live regions which are <code>assertive</code> will be presented immediately, followed by <code>polite</code> items. User agents or assistive technologies can choose to clear queued changes when an assertive change occurs. (e.g., changes in an assertive region can remove all currently queued changes)</p>
+				<p>When live regions are marked as <code>polite</code>, assistive technologies SHOULD announce updates at the next graceful opportunity, such as at the end of speaking the current sentence or when the user pauses typing. When live regions are marked as <code>assertive</code>, assistive technologies SHOULD immediately notify the user of modifications to the live region. Because an interruption might disorient users or cause them to not complete their current task, authors SHOULD NOT use the <code>assertive</code> value unless the interruption is imperative.</p>
+				<p>Typically, assistive technology will only convey <em>changes</em> to a live region, not the initial contents of a live region. To ensure content in a live region is announced, authors SHOULD create a rendered but empty live region as early as possible (such as on page load), and then modify the content of the live region when the author expects changes to be announced. The exception to this live region convention is <code>alert</code>, due to system accessibility notifications events required for the role. While an <rref>alert</rref> is a live region, its content is announced by assistive technology when the alert is rendered on the page and when the content changes.</p><!-- Duplicate paragraph in #live_region_roles. Synchronize any changes to these paragraphs. -->
```

Comment:
I'm not sure why GH isn't showing inline diffs here. I didn't change all that much. There's a new paragraph at the end, and the phrase "politeness" is now "priority in several places, along with a few other editorial changes.